frontend/public/components/factory/table.tsx

```diff
@@ -252,6 +252,13 @@ export type TableWrapperProps = {
   ariaRowCount: number;
 };
 
+class RowMemo extends React.PureComponent<RowFunctionArgs & { Row: React.FC<RowFunctionArgs> }> {
+  render() {
+    const { Row, ...props } = this.props;
+    return <Row {...props} />;
+  }
+}
```

Comment:
```suggestion
const RowMemo = React.memo(Row);
```

Reply:
I need to pass `Row` as prop, but sure, we can use React.memo instead of PureComponent. Updated to
```js
const RowMemo = React.memo<RowFunctionArgs & { Row: React.FC<RowFunctionArgs> }>(
  ({ Row, ...props }) => <Row {...props} />,
);
```